include/disccord/rest/client.hpp

```diff
@@ -11,12 +11,15 @@ namespace disccord
         {
             public:
                 discord_rest_client();
+                discord_rest_client(std::string token); //will default token type to 'Bot'
+                discord_rest_client(std::string token, disccord::token_type type); 
                 virtual ~discord_rest_client();
+                
+                disccord::rest::internal::rest_api_client get_client();
```

Comment:
heh yeah i figured we wouldn't want that, just added as a temp thing so I coud use this class instead.

 so you want a new set of methods that will make the requests with internal guy? if not that, not really sure how this will be wrapped tbh 🤔 but that seems like the cleanest thing to do.